templates/_helpers.yaml

```diff
@@ -154,8 +154,8 @@
 
 {{ define "pgbouncer_config" }}
 [databases]
-{{ .Release.Name }}-metadata = host={{ .Values.data.metadataConnection.host }} dbname={{ .Values.data.metadataConnection.db }} port={{ .Values.data.metadataConnection.port }} pool_size={{ .Values.pgbouncer.metadataPoolSize }}
-{{ .Release.Name }}-result-backend = host={{ .Values.data.resultBackendConnection.host }} dbname={{ .Values.data.resultBackendConnection.db }} port={{ .Values.data.resultBackendConnection.port }} pool_size={{ .Values.pgbouncer.resultBackendPoolSize }}
+{{ .Release.Name }}-metadata = host={{ .Values.data.metadataConnection.host }} dbname={{ .Values.data.metadataConnection.db }} port={{ .Values.data.metadataConnection.port }} pool_size={{ .Values.pgbouncer.metadataPoolSize }} {{ if .Values.data.metadataConnection.azureDbServer }}user={{ .Values.data.metadataConnection.user }}{{ .Values.data.metadataConnection.azureDbServer }} password={{ .Values.data.metadataConnection.pass }}{{ end }}
```

Comment:
@ianstanton is there a way we could standardize to a format that works for typical behavior and azure behavior, rather than adding `user` and `password` _only_ for azure? Could we do that for both?

Reply:
@ianstanton I'm hoping that this would also let us drop `azureDbServer` out of houston, keeping the logic generic and consolidated.